gallery/plot_transforms_v2_e2e.py

```diff
@@ -0,0 +1,147 @@
+"""
+==================================================
+transforms v2: End-to-end object detection example
```

Comment:
I don't know how to keep it short, but it'd be nice to emphasize in the title that one can now jointly transforms images/boxes/masks.

Reply:
Maybe not in the title, but in the description just below the title? Something along the lines: 

> Object detection is not supported out of the box by transforms v1, since it only supports images. transforms v2 enables jointly transforming amongst others images, bounding boxes, and masks. This example showcases an end-to-end object detection training using stable torchvision datasets and models as well as the new tranforms v2 API.

Reply:
I went for my suggestion in the latest commit. LMK if you are ok with that.